test/interp/simd-binary.txt

```diff
@@ -130,6 +130,26 @@
     v128.const i32 0x00ffffff 0x0400ffff 0x00000003 0x00000004
     v128.const i32 0x00020001 0xfe000002 0x00000003 0x00000004
     i16x8.add_saturate_u)
+
+  ;; i8x16 saturating sub (signed and unsigned)
+  (func (export "i8x16_sub_saturate_signed_0") (result v128)
+    v128.const i32 0x00000001 0x0000007f 0x00000003 0x00000080
+    v128.const i32 0x00000001 0x00000002 0x00000003 0x000000ff
+    i8x16.sub_saturate_s)
+  (func (export "i8x16_sub_saturate_unsigned_0") (result v128)
+    v128.const i32 0x00ff0001 0x04000002 0x00000003 0x00000004
+    v128.const i32 0x00020001 0xfe000002 0x00000003 0x00000004
```

Comment:
`0x4 - 0xfe` should be `0`, right? I think this is happening because the arithmetic is done with `uint32_t`, which will wrap to `0xffffff06` before being clamped to `0xff`.

Reply:
@binji 
Yes, you are right. Should always keep result value as signed value for Saturating Sub operation.